web/Services/PropertyService.cs

```diff
@@ -160,12 +167,12 @@ public bool IsUserAdminOrCorrectLandlord(BricksAndHeartsUser currentUser, int pr
             return true;
         }
 
-        var propertyLandlordId = GetPropertyByPropertyId(propertyId)!.LandlordId;
-        var userLandlordId = currentUser.LandlordId;
-        return propertyLandlordId == userLandlordId;
+        return currentUser.LandlordId != null &&
+               GetPropertyByPropertyId(propertyId)?.LandlordId == currentUser.LandlordId;

```

Comment:
can we move the `&&` operator onto the start of the next line please :) it makes it easier to read

Reply:
Same as above :)